config/default.yml

```diff
@@ -689,13 +689,17 @@ Naming/UncommunicativeBlockParamName:
   AllowNamesEndingInNumbers: true
   # Whitelisted names that will not register an offense
   AllowedNames: []
+  # Blacklisted names that will register an offense
+  ForbiddenNames: []
```

Comment:
You should probably mention this option in the cop documentation as well. I assume more people read the manual than the config file. :-)